azkaban-common/src/main/java/azkaban/utils/RestfulApiClient.java

```diff
@@ -222,20 +226,20 @@ public T httpDelete(final URI uri, final List<NameValuePair> headerEntries) thro
    *
    * @param uri the URI of the request.
    * @param headerEntries extra entries to be added to request header.
-   * @param postingBody the content to be posted , optional.
+   * @param params the content to be posted , optional.
    * @return the response object type of which is specified by user.
    * @throws UnsupportedEncodingException, IOException
    */
   public T httpPut(final URI uri, final List<NameValuePair> headerEntries,
```

Comment:
This method isn't used for anything though. Should it be deleted, or keep for "completeness"?

Reply:
Please help to delete dead code!

Thanks!